include/http/request_handler.js

```diff
@@ -164,9 +164,9 @@ module.exports = function RequestHandlerModule(pb) {
           uid: site.uid,
           displayName: site.displayName,
           hostname: site.hostname,
-          defaultLocale: site.defaultLocale,
-          supportedLocales: site.supportedLocales,
-          prevHostnames: site.prevHostnames
+          defaultLocale: site.defaultLocale || pb.Localization.getDefaultLocale(),
+          supportedLocales: site.supportedLocales || [],
```

Comment:
@wesanderson303 Are you sure this works as expected?  I *think* this will always default to English because there are no other allowed locales for the site